ext/php_driver.h

```diff
@@ -117,6 +117,9 @@ typedef zend_object php5to7_zend_object_free;
 typedef zval **php5to7_zval_gc;
 typedef zval *php5to7_dtor;
 typedef size_t php5to7_size;
+#if PHP_MAJOR_VERSION >= 7 && PHP_MINOR_VERSION >= 4
```

Comment:
Thanks for the fix! I stumble upon this bug as well when manually compiling (which is necessary for 7.3+).
However, this PR would only fix this for PHP ^7.4 and not 8.0 or above.
Why not change line 120 to

#if ((PHP_MAJOR_VERSION == 7 && PHP_MINOR_VERSION >= 4) || PHP_MAJOR_VERSION > 7)

Reply:
Thanks for the hint, I changed the PR as you suggested.